src/impls.rs

```diff
@@ -346,6 +346,15 @@ impl Format for () {
     }
 }
 
+impl<T> Format for core::marker::PhantomData<T> {
+    fn format(&self, f: Formatter) {
+        if f.inner.needs_tag() {
+            let t = internp!("PhantomData<_>");
```

Comment:
could you change the string to `PhantomData` so that this resembles how `core::fmt::Debug` formats this type?